nxdrive/commandline.py

```diff
@@ -367,7 +367,7 @@ def parse_cli(self, argv: List[str]) -> Namespace:
         has_command = False
 
         # Pre-configure the logging to catch early errors
-        configure(console_level="DEBUG", command_name="early")
+        configure(console_level="INFO", command_name="early")
```

Comment:
This one should be kept at `DEBUG`, this is the time we initialize the logging and to being able to catch option updates.